jsh/db/migrations/20230515131608_users.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+create table comments (
+  id int not null auto_increment,
+  content varchar(3000) not null,
+  user_id int not null,
+  post_id int not null,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+    PRIMARY KEY(id),
+  FOREIGN KEY (user_id) REFERENCES users(id),
+  FOREIGN KEY (post_id) REFERENCES posts(id)
+)
```

Comment:
코드의 가독성을 높이는 방법에는 다양한 방법이 있습니다. 

- 다른 `sql` 파일을 확인해보면 대소문자가 통일되어 있는 것 같습니다. 해당 파일에서도 통일시켜 주세요!
- 외래키를 설정할 때 추후 제약 조건의 이름을 통해 관리할 수 있도록 `CONSTRAINT ... ` 구문을 사용해주세요!

```suggestion
CREATE TABLE comments (
  id INT NOT NULL AUTO_INCREMENT,
  content VARCHAR(3000) NOT NULL,
  user_id INT NOT NULL,
  post_id INT NOT NULL,
  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY(id),
  CONSTRAINT comments_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id),
  CONSTRAINT comments_post_id_fkey FOREIGN KEY (post_id) REFERENCES posts(id)
)
```